.kitchen.yml

```diff
@@ -9,13 +9,68 @@ platforms:
   - name: ubuntu-14.04
     run_list:
     - "recipe[apt::default]"
+  - name: centos-7.1-vbox
```

Comment:
Then all systems even a ubuntu or windows system would need to have a yum cookbook.  It creates bloat that is not needed.  The cookbook itself will determine if the dependency is needed based on the execution of the cookbook.  Ideally the depends should have a os check on it.